next-app/src/products/product-types.ts

```diff
@@ -12,6 +12,14 @@ export type Product = {
   title: string;
 };
 
+export type Chemical = {
```

Comment:
no need for image, assume image is under "/public/images/chemicals/[chemicalID]". Ok to keep for now, but may remove in future to be dynamic